src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -104,87 +119,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /* Store Scavenger Hunt object in Datastore. */
```

Comment:
nit: Mention in javadoc that the return value is the key of the new datastore entry.